lib/active_snapshot/config.rb

```diff
@@ -0,0 +1,20 @@
+class ActiveSnapshot::Config
+  class << self
+    attr_accessor :storage_method
```

Comment:
lets use attr_reader instead I want the writer method to enforce the list of possible options.